README.md

```diff
@@ -28,29 +28,20 @@ developments that cannot be integrated into core TensorFlow
 ## Maintainers
 | Subpackage    | Maintainers  | Contact Info                        |
 |:----------------------- |:----------- |:----------------------------|
-| tfa.activations | SIG-Addons | addons@tensorflow.org    |
-| tfa.image |  |                                   |
-| tfa.layers | SIG-Addons |     addons@tensorflow.org |
-| tfa.losses | SIG-Addons |     addons@tensorflow.org |
-| tfa.optimizers | SIG-Addons | addons@tensorflow.org |
-| tfa.seq2seq | Google | @qlzh727 | 
-| tfa.text |  |  |
+| [tfa.activations]((tensorflow_addons/activations/README.md)) | SIG-Addons | addons@tensorflow.org    |
```

Comment:
Please remove the extra '(' and ')', the link is broken

Reply:
Good catch, thanks!